stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py

```diff
@@ -31,6 +31,13 @@ class PutItem(ItemUri):
     item: stac_types.Item = attr.ib(default=Body(None))
 
 
+@attr.s
+class PutCollection(CollectionUri):
+    """Update Collection."""
+
+    collection: stac_types.Collection = attr.ib(default=Body(None))
```

Comment:
I'm not sure to understand why we default to `Body(None)` here

Reply:
The PutItem class above does the same thing which is maybe why it was done here?

Reply:
Indeed, I did not give that any thought whatsoever. 🙈